http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1781,6 +2082,57 @@ components:
         patchInProgress:
           type: boolean
           description: Whether there is a patch in progress on the runtime. Is used to indicate updates that require status transitions.
+    PersistentDisk:
+      description: ""
+      required:
+        - id
+        - googleProject
+        - zone
+        - name
+        - googleId
+        - samResourceId
```

Comment:
are we not return `error` for  GetPersistentDisk route?...I don't think `labels` is required...
Not all these fields are applicable to `listPersistentDiskResponse` I think.

Reply:
We don't have errors for PDs yet, I think we were holding off until we have a better idea of what kinds of errors can come up. @rtitle would that be after some of the back Leo work?

Reply:
Yeah we haven't added any disk errors yet… let's hold off till another PR, I don't want to keep increasing scope of this one.